cppcache/integration-test/testThinClientCqDurable.cpp

```diff
@@ -192,7 +192,7 @@ void RunDurableCqClient() {
   // Create a Geode Cache Programmatically.
   auto cacheFactory = CacheFactory(pp);
   auto cachePtr = std::make_shared<Cache>(cacheFactory.create());
-  auto poolFactory = cachePtr->getPoolManager().createFactory();
+  auto poolFactory = std::make_shared<PoolFactory>(cachePtr->getPoolManager().createFactory());
```

Comment:
Is `shared_ptr` necessary?

Reply:
Not necessary. Just minimal change to get the code working.  Probably best to keep test code up-to-date so I'll see about making suggested changes to remove `shared_ptr` in this context in integration-tests.